docs/notebooks/survival_model.ipynb

```diff
@@ -0,0 +1,1196 @@
+{
```

Comment:
<p>I would add one or two sentences about the <code>censored()</code> function, which you can find here <a href="https://github.com/bambinos/bambi/blob/65bbae4faa70461a2b3983a72965b4fb337de3d9/bambi/transformations.py#L12" target="_blank">https://github.com/bambinos/bambi/blob/65bbae4faa70461a2b3983a72965b4fb337de3d9/bambi/transformations.py#L12</a>. Basically, I would say it takes two arguments. The first is the observed value, and the second is "left", "none", or "right" indicating left, none, or right censoring. I wouldn't get into interval censoring as it doesn't work very well out of the box in PyMC. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/742/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Definitely agree. Thanks!</p><p> </p>